src/WebJobs.Script/DependencyInjection/ScriptStartupTypeLocator.cs

```diff
@@ -84,18 +84,17 @@ public async Task<IEnumerable<Type>> GetExtensionsStartupTypesAsync()
             bool isPrecompiledFunctionApp = false;
 
             // dotnet app precompiled -> Do not use bundles
-            var workerConfigs = _languageWorkerOptions.CurrentValue.WorkerConfigs;
```

Comment:
The `IOptionsMonitor<LanguageWorkerOptions>` dependency is nolonger used in this class. Should we consider removing it from being injected?